src/internal/styles/direction/index.scss

```diff
@@ -0,0 +1,6 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@forward './mixins';
```

Comment:
Do we really need multiple layers of abstract files which will never come useful?

Reply:
I was just following the existing pattern. No opinion here. I can flatten it if you prefer.

Reply:
Yes please. The forward in `src/internal/styles/index.scss` makes sense, but this one does not. It is only useful when you have multiple files to forward